### DISCUSSION TITLE
No "swa" in my path after installing!

### DISCUSSION THREAD

**Discussion by siegfried01** [Azure/static-web-apps-cli#588] [Q&A]
I am following along with [https://learn.microsoft.com/en-us/azure/static-web-apps/add-api?tabs=angular#start-the-cli](url) and I run the command

     npm install -g @azure/static-web-apps-cli

but there is no "swa" in my path. 

     swa: The term 'swa' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Where is swa?

Thanks
Siegfried

**Comment by mariacheline** [comment]
+1
